server/build_event_protocol/build_event_handler/build_event_handler.go

```diff
@@ -471,7 +472,7 @@ func (e *EventChannel) FinalizeInvocation(iid string) error {
 		return err
 	}
 
-	e.statsRecorder.Enqueue(e.ctx, invocation)
+	e.statsRecorder.Enqueue(ctx, invocation)
```

Comment:
I think this is a good change, so this is more just intended as a note: `Enqueue` only uses the context for the purpose of inspecting its values (`x-buildbuddy-jwt`), so I think the previous code would have worked even if the ctx deadline was exceeded

Reply:
Good to know! Thank you for the info, as always ^-^